DigitalLearningSolutions.Web/Views/NotificationPreferences/Index.cshtml

```diff
@@ -34,6 +52,25 @@
           </span>
         </summary>
         <div class="nhsuk-details__text">
+          <dl class="nhsuk-summary-list preference-list">
+            @foreach (var notification in Model.DelegateNotifications)
+            {
+              <div class="nhsuk-grid-row preference-row">
+                <div class="nhsuk-grid-column-three-quarters">
+                  <div class="nhsuk-heading-xs">@notification.NotificationName</div>
+                  <p>@Html.Raw(notification.Description)</p>
+                </div>
+                <div class="nhsuk-grid-column-one-quarter">
+                  <a class="preference-button nhsuk-button @(notification.Accepted ? "subscription-button--active" : "nhsuk-button--secondary")" role="button">

```

Comment:
Could we make these tags as the user won't be able to interact with it - https://service-manual.nhs.uk/design-system/components/tag

Here are some colours: 
https://service-manual.nhs.uk/design-example/tag-colours?fullpage=undefined&blankpage=undefined

We could use grey for unsubscribed and either blue or green for subscribed? You could post screenshots to the external HEE channel and ask which version HEE prefers :)

Reply:
Oup, I had been assuming the user would be clicking on these in some later ticket to subscribe and unsubscribe and using the Update Preferences button to send the request, but this makes more sense. I'll fix it.

Reply:
Yeah - unfortunately there's no good way of showing the tags in the wireframes, so it looks really similar to buttons 😢